cypress/features/regression/test/badge.feature

```diff
@@ -1,45 +1,44 @@
 Feature: Badge component
   I want to test Badge component properties
 
-  Background: Open Badge component page
-    Given I open basic Test "Badge" component page
-
   @positive
   Scenario Outline: Set Badge component to <counter>
-    When I set counter to "<counter>"
+    When I open Test basic "Badge" component in noIFrame with "badge" json from "test" using "<nameOfObject>" object name
     Then Badge component counter is set to <counter>
     Examples:
-      | counter |
-      | 1       |
-      | 99      |
+      | counter | nameOfObject |
+      | 1       | counter1     |
+      | 99      | counter99    |
 
   @positive
   Scenario Outline: Set Badge component to 3 digits to <counter>
-    When I set counter to "<counter>"
+    When I open Test basic "Badge" component in noIFrame with "badge" json from "test" using "<nameOfObject>" object name
     Then Badge component counter is set to 99
     Examples:
-      | counter |
-      | 100     |
-      | 999     |
+      | counter | nameOfObject |
+      | 100     | counter100   |
+      | 999     | counter99    |
 
   @negative
   Scenario Outline: Set Badge component to out of scope to <counter>
-    When I set counter to <counter> word
+    When I open Test basic "Badge" component in noIFrame with "badge" json from "test" using "<nameOfObject>" object name
     Then Badge component counter does not exist
     Examples:
-      | counter                      |
-      | 0                            |
-      | -12                          |
-      | test                         |
-      | !@#$%^*()_+-=~[];:.,?{}&"'<> |
+      | counter                 | nameOfObject            |
+      | 0                       | counter0                |
+      | -12                     | counter-12              |
+      | test                    | counterTest             |
+      | !@$^*()_%2B-=~[];:.,?{} | counterSpecialCharacter |
 
   @positive
   Scenario: Hover Badge component and verify that cross icon appears
+    Given I open basic Test "Badge" component page
     When I focus onto Badge component
     Then icon on preview is "cross"
 
   @positive
   Scenario: Click event
```

Comment:
move this TC to separate feature file as you did with `alert` component , please

Reply:
fixed